core/pom.xml

```diff
@@ -487,9 +487,9 @@ THE SOFTWARE.
       <version>20110809</version>
     </dependency>
     <dependency>
-      <groupId>commons-collections</groupId>
-      <artifactId>commons-collections</artifactId>
-      <version>3.2.1</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
Ok. Should I add version 4 next to version 3 then? They have a different classpath (org.apache.commons.collections vs org.apache.commons.collections4) so I don't think they'd conflict.